wordpress/var/www/html/wp-config.php.ctmpl

```diff
@@ -12,10 +12,11 @@
  * The following database info is needed here, but is ignored after we load HyperDB.
  * See db-config.php.ctmpl for details.
  */
-define( 'DB_NAME',     '{{key "wordpress/config/DB_NAME"}}' );
-define( 'DB_USER',     '{{key "wordpress/config/DB_USER"}}' );
-define( 'DB_PASSWORD', '{{key "wordpress/config/DB_PASSWORD"}}' );
-define( 'DB_HOST',     '{{with index (service "dbmaster") 0}}{{.Address}}:{{.Port}}{{end}}' );
+ ## this is a test
+define( 'DB_NAME',     '{{env "MYSQL_DATABASE"}}' );
+define( 'DB_USER',     '{{env "MYSQL_USER"}}' );
+define( 'DB_PASSWORD', '{{env "MYSQL_PASSWORD"}}' );
+define( 'DB_HOST',     '{{range service "mysql-primary"}}{{.Address}}:{{.Port}}{{end}}' );
```

Comment:
I struggled to find the right syntax that would let me select just _one_ entry here, but I'm pretty sure this would result in an invalid config if there were multiple masters. It would be a very bad thing :tm: if there were multiple masters, but I'd prefer to reduce the number of things that would blow up.